src/js/pendefine/nagenawa.js

```
@@ -0,0 +1,503 @@
+        if (this.state === 'drawing' && this.axpObj.isDrawing && !this.axpObj.isDrawCancel) {
+            this.axpObj.isDrawn = true;
+            this.lassoPath.push({ x, y });
+            this.drawLassoTrail();
+        } else if (this.state === 'transforming' && this.axpObj.isDrawing && !this.axpObj.isDrawCancel) {
+            this.affine.tx = this.affineSaved.tx + (x - this.dragStartX);
+            this.affine.ty = this.affineSaved.ty + (y - this.dragStartY);
+            this.drawTransformed(false);
+        }
+    }
+    // 描画終了
+    end(x, y, e) {
```

ESLint

[error] 'x' is defined but never used.

(no-unused-vars)